user_home/templates/base.html

```diff
@@ -37,17 +37,38 @@
         <div class="col-md-4">
           <div class="content-section">
               <ul class="list-group">
+
+<!--                  Tier-1 menu-->
                   <li class="list-group-item list-group-item-light menu" id="fundTransfer">
-                      <a href="http://localhost:8000/internal_user/initFundDeposit">Initiate fund deposit</a> </li>
+                      <a href="http://{{ request.get_host }}/internal_user/initFundDeposit">Initiate fund deposit</a> </li>
                   <li class="list-group-item list-group-item-light menu" id="issueCheque">
-                      <a href="http://localhost:8000/internal_user/initIssueCheque">Issue cashier cheque</a> </li>
+                      <a href="http://{{ request.get_host }}/internal_user/initIssueCheque">Issue cashier cheque</a> </li>
                   <li class="list-group-item list-group-item-light menu" id="transferFund">
-                      <a href="http://localhost:8000/transaction/initfundTransfer">Transfer money</a> </li>
+                      <a href="http://{{ request.get_host }}/transaction/initfundTransfer">Transfer money</a> </li>
+
+                  <!--    Tier-1 and Tier-2 common menu-->
                   <li class="list-group-item list-group-item-light menu" id="viewCustomerAccount">
-                      <a href="http://localhost:8000/internal_user/initViewCustomer">View customer account</a> </li>
+                      <a href="http://{{ request.get_host }}/internal_user/initViewCustomer">View customer account</a> </li>
                   <li class="list-group-item list-group-item-light menu" id="pendingTransactions">
-                      <a href="http://localhost:8000/transaction/pendingTransactions">Pending Transactions</a> </li>
-              </ul>
+                      <a href="http://{{ request.get_host }}/transaction/pendingTransactions">Pending Transactions</a> </li>
+
+<!--                  Tier-2 menu-->
```

Comment:
Are the menu options for all tiers going to be shown for all internal users? Because we derive from base.html for all other tier users.. ?

Reply:
Menu will be different based on user type. This change is still pending. The current user object in login view does not have user type attribute.